rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,151 @@
+
+# Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas that have sparked interest of commenters
+in their order of submission. They shall be considered _optional_ material
+during implementation, whereas implementors shall have the liberty to
+deliver a consistent experience as they see fit.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can _still_ \* maintain those high quality direct human
+interactions, while having full support of a _"curriculum-like"_ resource.
+
+\* _still_, as an aside, because nix community is known for it's great
+mentorship spirit.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There _can_ be a better solution to the achieve _optimum levels of empowerment_
+for our community. We _could just **wait**_ for them to unveil themselves.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Benevolent Dictatorship
+- Ephimeral Conversations (Discourse)
+
+The only current viable alternative for _coordination of resources and effort_
+are ephimeral conversations in Discourse. While, they lend meaning to the 
+individual's interpretations and views onto the ecosystem, they lack important
+features such as:
+
+- Structure
+- Consolidation
+- Ability to act as _Reference_
+
+A portal page might at first sight seem like an alternative, especially for those
+features. But, _optimum empowerment_ requires low barriers to interaction and
+amendment. Hence a git repository. Repository tooling can complement this aspect
+and deploy a static page under the nixos.org for a design lift on the content.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Is this the right level of abstraction to act as an enabling RFC for future
+improvements and thereby resume the author's stance in a way that effects
+the highest leverage over time?
+
+# Future work
+[future]: #future-work
+
+- Constitution of Woring Groups, such as (examplifications):
+  - WG Contribution Workflows
+  - WG Onboarding
+- Constitution of Special Interest Groups (examplifications):
+  - SIG Rust
+  - SIG Security
+  - SIG Golang
+  - SIG Python
+  - SIG Contributor Experience
+  - SIG Command Line Interface "nix 2.0"
+  - SIG Tooling & Automation
+  - SIG Testing & Infrastructure
+- Constitution of User Groups (examplifications):
+  - UG NixOS
+  - UG Flakes
+
+(those are way too many, of course)
```

Comment:
Why? :)

Reply:
It's not the purpose of this RFC to establish consensus on possible future actions. So those are just envelop hedging words.

Reply:
I attenuated so as not to make it a stance.

Reply:
:+1:.